modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -193,10 +202,16 @@ export SPACEWALK_USER='<adminname>';export SPACEWALK_PASS='<password>'
 ----
 ln -s /usr/local/bin/cent-errata.sh /etc/cron.daily
 ----
-. Open the ``rhn.conf`` configuration file, and add or edit this line:
-+
-----
-java.allow_adding_patches_via_api = centos7-x86_64-updates,centos7-x86_64,centos7-x86_64-extras
-----
 
 For more information on this tool, see https://cefs.steve-meier.de/.
+
+[WARNING]
```

Comment:
Can we move this above the procedure? There is an `important` admon on L144, and I think we can probably combine them.

Reply:
Makes sense, addressed.